packages/meditrak-app/ios/TupaiaMediTrak/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.11.121</string>
+	<string>1.12.123</string>
```

Comment:
Funny new app number here, it seems like we track the version just by the last number? But semver suggests we should bump the middle number for new features...

Reply:
Yes it is funny! Have just updated the slab doc [here](https://beyond-essential.slab.com/posts/releasing-medi-trak-app-via-appcenter-psgr6iak) to explain the reason, but to repost:

> Don't reset or wrap patch number, instead ALWAYS just increment it by 1 with every release.  This is because we use a shortcut where the patch number doubles as the android build number, which needs to increase monotonically.